src/util/metadata/side_metadata/mod.rs

```diff
@@ -22,3 +22,9 @@ pub(crate) use helpers::*;
 #[allow(unused_imports)]
 pub(crate) use helpers_32::*;
 pub(crate) use sanity::SideMetadataSanity;
+
+/// Side metadata spec for VO bit. This is only available when the feature "vo_bit_access" is enabled.
+/// Check the comments on "vo_bit_access" in `Cargo.toml` before use. Use at your own risk.
+#[cfg(feature = "vo_bit_access")]
+pub const VO_BIT_SIDE_METADATA_SPEC: SideMetadataSpec =
```

Comment:
Once we make the `vo_bit` module public, we can move this into the `vo_bit` module directly, as a `pub const`.